CHANGES.rst

```diff
@@ -290,6 +290,17 @@ astropy.io.fits
 astropy.io.misc
 ^^^^^^^^^^^^^^^
 
+Bug fix
```

Comment:
This need to be cleanup. Please don't add title lines, but rather add the entries directly to under the correct heading.